Overhaul

tacerus opened the pull request:
Reference the commit messages for details.

tacerus committed:
Update Ruby stack

Update pinned Gems and update development build instructions for
current Ruby stack on Tumbleweed. The additional packages seem to
be required for building the dependencies - it might not have been
noticed on standard installations before as they are required by many
commonly installed packages, but building in a container with minimal
packages revealed them not being pulled by the other, already
explicitly mentioned, packages.

Signed-off-by: Georg Pfuetzenreuter <mail@georg-pfuetzenreuter.net>

cboltz commented on index.html:
```diff
@@ -4,30 +4,32 @@
-        , but enhance our setup with some additional packages from the 
-        <a href="https://build.opensuse.org/project/show/server:monitoring" title="server:monitoring Repo" target="_blank">server:monitoring</a> 
+        The setup uses packages from our preferred <a href="https://www.opensuse.org">distribution</a>
+        , and is enhanced with custom packages gathered in the
```
Nitpick: please move the comma directly after the link to avoid whitespace.

tacerus replied:
Ok!

tacerus committed:
Update copyright, description, license

- Bump copyright year to reflect 2024
- Improve grammar in description
- Repair link to license file

Signed-off-by: Georg Pfuetzenreuter <mail@georg-pfuetzenreuter.net>

tacerus committed:
Overhaul

- explain move to Prometheus
- replace deprecated Icinga frontend with new alerting page
- reference our custom OBS project
- link dashboard overview instead of individual Grafana dashboards

Signed-off-by: Georg Pfuetzenreuter <mail@georg-pfuetzenreuter.net>

tacerus committed:
Update IRC section

- use irc.libera.chat directly instead of the CNAME which
  comes without TLS and with a dependency on our infrastructure
- simplify logging advisory
- reference new alerts channel

Signed-off-by: Georg Pfuetzenreuter <mail@georg-pfuetzenreuter.net>
